service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsController.java

```diff
@@ -161,20 +164,26 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(@RequestBod
 
         ManageDocumentType manageDocumentType = caseData.getManageDocument().getType();
         List<Element<SupportingEvidenceBundle>> currentBundle;
+        Set<DocumentWithConfidentialAddress> documentWithConfidentialAddress;
```

Comment:
Why are we using a set rather than a list?

Reply:
removed this unused variable.